addons/account/models/chart_template.py

```diff
@@ -329,6 +314,28 @@ def tax_template_changed(tax, template):
                 ('module', '=', 'account'),
             ]).unlink()
 
+        custom_fields = {  # Don't alter values that can be changed by the users
+            'account.fiscal.position.tax_ids',
+        }
+        for model_name, records in data.items():
+            _fields = self.env[model_name]._fields
+            for xmlid, values in records.items():
+                x2manyfields = [
+                    fname
+                    for fname in values
+                    if fname in _fields
+                    and f"{model_name}.{fname}" not in custom_fields
+                    and _fields[fname].type in ('one2many', 'many2many')
+                    and isinstance(values[fname], (list, tuple))
+                ]
+                if x2manyfields:
+                    rec = self.ref(xmlid, raise_if_not_found=False)
+                    if rec:
+                        for fname in x2manyfields:
+                            for i, (line, (command, _id, vals)) in enumerate(zip(rec[fname], values[fname])):
+                                if command == Command.CREATE:  # converts ORM command `create` into `update`
```

Comment:
Not what I asked for but agreed on discord that the comment could lie if more processing stuff are added to the loop later on.